libr/core/cmd_type.c

```diff
@@ -25,8 +25,17 @@ static void show_help(RCore *core) {
 }
 
 static int sdbforcb(void *p, const char *k, const char *v) {
-	if(!strncmp(k,"type.",strlen("type.")))
+	if(!strncmp(k,"type.",5))
 		r_cons_printf ("%s\n", k+5);
+	else if(!strncmp(k,"struct.",7)){
```

Comment:
strlen(const char*) is computed at compile time, the final binary will not have the strlen call just 5. but yeah its kind of implicit